cangjie5.dict.yaml

```diff
@@ -30640,8 +30640,8 @@ encoder:
 䛵	yrbbe
 㙜	yrbg
 䯨	yrbk
-䯩	yrbu
 䛁	yrbmm
+䯩	yrbu
```

Comment:
`䯩	yrbu` should be placed after `䛁	yrbmm` according to Cangjie code order. No observable changes because there is only one `yrbu` in Ext.  A.